target/linux/octeon/image/Makefile

```diff
@@ -29,6 +29,15 @@ define Device/generic
 endef
 TARGET_DEVICES += generic
 
+ITUSROUTER_CMDLINE:=console=ttyS0,115200, root=/dev/mmcblk1p2 rootfstype=squashfs,ext4,f2fs rootwait
```

Comment:
Is the comma after 115200 necessary/intended?

Reply:
Will remove

Reply:
Removed

Reply:
it's still there?

Reply:
Fixed.. not sure what happened